roaring/roaring.go

```diff
@@ -3335,9 +3382,16 @@ func popcount(x uint64) uint64 {
 }
 
 func popcountAndSlice(s, m []uint64) uint64 {
+	var (
+		a = s[:bitmapN]
+		b = m[:bitmapN]
+	)
+	_ = a[bitmapN-1]
+	_ = b[bitmapN-1]
```

Comment:
in my testing, this code is consistently faster without these two `_ =...` lines. See jaffee/bounds-check - I ran it on my macbook and on an Azure VM